cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4MeasureReportBuilder.java

```diff
@@ -348,21 +348,21 @@ protected void buildGroup(
 
             // add extension to group for
             if (bc.measureReport.getType().equals(MeasureReport.MeasureReportType.INDIVIDUAL)) {
-                String doc = null;
-                if (getReportPopulation(groupDef, DATEOFCOMPLIANCE) != null
-                        && !getReportPopulation(groupDef, DATEOFCOMPLIANCE)
-                                .getResources()
-                                .isEmpty()) {
-                    doc = getReportPopulation(groupDef, DATEOFCOMPLIANCE)
-                            .getResources()
-                            .iterator()
-                            .next()
-                            .toString();
+                var docPopDef = getReportPopulation(groupDef, DATEOFCOMPLIANCE);
+                if (docPopDef != null
+                        && docPopDef.getResources() != null
+                        && !docPopDef.getResources().isEmpty()) {
+                    Interval docInterval =
+                            (Interval) docPopDef.getResources().iterator().next();
```

Comment:
I think you should do an instanceof check and handle the error if it fails.  A ClassCastException can be an ugly error to try to debug